src/utils/searchTermOption.ts

```
@@ -1,65 +1,111 @@
+  name: string;
+  id: string;
+  endDate: string | null;
 };
 
-export type SearchLabel = keyof typeof searchFn;
+// 검색 함수 맵
+//물품,공간,유저,이용현황 검색어 찾기
+const searchFn: Record<
+  (typeof SEARCH_TERMS)[keyof typeof SEARCH_TERMS],
+  () => Promise<{
+    data: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)